docs/self-managed/components/management-identity/configuration/connect-to-an-oidc-provider.md

```diff
@@ -138,6 +96,17 @@ Once set, you cannot update your initial claim name and value using environment
 
 For authentication, the Camunda components use the scopes `email`, `openid`, `offline_access`, and `profile`.
 
+:::tip Optional scopes
+The `offline_access` scope is optional.
```

Comment:
Here we could explain a bit what the purpose of the `offline_access` scope is, so that users can decide more easily if they set it or not.

Something like this:

If the `offline_access` scope is present, your OIDC provider will issue a refresh token to Management Identity on user login. Management Identity will then use the refresh token to renew the user's access token when it expires, so that the user's session is automatically extended. Without the scope present, when the access token expires the user will be redirected to the OIDC provider for new authentication.

And we could link to https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccess as the specification reference.